EtherealEngine/etherealengine
# behave graph patch package

## Summary

## References

## Explanation

## QA Steps
- `rm -rf node_modules/@behave-graph &&  rm -rf package-lock.json && npm i`
- run `npm run dev` and ensure it loads any location correctly


## Checklist
- If this PR is still a WIP, convert to a draft
- When this PR is ready, mark it as "Ready for review"
- [ensure all checks pass](https://github.com/etherealengine/etherealengine/wiki/Testing-&-Contributing)
- Changes have been manually QA'd 
- Changes reviewed by at least 2 approved reviewers
